CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
+## 8.6.0 - December 19, 2019
+### Bug fixes
+ - Revert of broken cycle debug options [#16074](https://github.com/mapbox/mapbox-gl-native/pull/16074)
+ - Fixed a crash caused by an unintentional minification of the `LocalGlyphRasterizer`. [#102](https://github.com/mapbox/mapbox-gl-native-android/pull/102)
+ - Offline download must clear batch buffer at load interrupt [#16074](https://github.com/mapbox/mapbox-gl-native/pull/16074)
```

Comment:
Maybe we can add more color? "Fixed intermittent offline download failures by ..."?

Reply:
I like this suggestion